test/dynamo/test_misc.py

```diff
@@ -7631,6 +7631,58 @@ def my_dyn_fn(x):
 
         torch._dynamo.optimize("eager")(my_dyn_fn)(y)
 
+    @torch._dynamo.config.patch(specialize_float=False, capture_scalar_outputs=True)
+    def test_unspecialized_float_multiply_precision_bf16(self):
```

Comment:
nit: It might be good to move all dynamic related tests to their own file, maybe  dynamo/test_dynamic.py